code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -204,8 +204,10 @@
 		/obj/item/flashlight/pen,
 		/obj/item/seeds,
 		/obj/item/wirecutters,
-        /obj/item/wrench,
-	)
+		/obj/item/wrench,
+		/obj/item/reagent_containers/spray/weedspray,
+		/obj/item/reagent_containers/spray/pestspray
+        )
```

Comment:
```suggestion
		)
```
We indent with tabs here

Reply:
i.. have no words. i SWEAR i indented with tabs originally. this pr has a curse on it 😒